src/LiveChat/Api/Rest/RestRequest.php

```diff
@@ -62,6 +64,16 @@ public function setPassword($password)
     }
 
     /**
+     * Set proxy
+     * @param string $proxy Url:Port
+     */
+    public function setProxy($proxy)
+    {
+        $this->proxyPort = parse_url( $proxy, PHP_URL_PORT ) ?: 80;
+        $this->proxy = str_replace( ':' . $this->proxyPort, '', $proxy );

```

Comment:
Code formatting: please remove empty spaces before first and after last functions arguments.